examples/text-generation/run_generation.py

```diff
@@ -114,6 +114,21 @@ def main():
         type=str,
         help="Optional argument to give a prompt of your choice as input.",
     )
+    parser.add_argument(
+        "--bad_words_ids",
```

Comment:
I think we should rename this arg to `--bad_words` actually, since IDs are obtained after encoding with the tokenizer.

Reply:
Done